File: rxwebview/src/androidTest/java/com/github/satoshun/reactivex/webkit/RxWebViewTestActivity.java
package com.github.satoshun.reactivex.webkit;

import android.app.Activity;
import android.os.Bundle;
import android.webkit.WebView;
import android.widget.FrameLayout;

public class RxWebViewTestActivity extends Activity {
  FrameLayout parent;
  WebView webview;

  @Override protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    parent = new FrameLayout(this);
    webview = new WebView(this);
    parent.addView(webview);
    setContentView(parent);
  }
}

File: rxwebview/src/androidTest/java/com/github/satoshun/reactivex/webkit/RxWebViewTestActivity.kt
package com.github.satoshun.reactivex.webkit

import android.app.Activity
import android.os.Bundle
import android.webkit.WebView
import android.widget.FrameLayout

class RxWebViewTestActivity : Activity() {
  private lateinit var parent: FrameLayout
  lateinit var webView: WebView

  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    parent = FrameLayout(this)
    webView = WebView(this)
    parent.addView(webView)
    setContentView(parent)
  }
}
